draft-ietf-core-dns-over-coap.md

```diff
@@ -275,6 +278,63 @@ The construction algorithm for DoC requests is as follows, going through the pro
 
 A more generalized construction algorithm for any CoAP request can be found in {{-transport-indication}}.
 
+### Examples
+
+A typical SVCB resource record response for a DoC server at the root path "/" of the server looks
+like the following (the "docpath" SvcParam is the last 4 bytes "00 0a 00 00" in the binary):
+
+    Resource record (binary):
+      04 5f 64 6e 73 07 65 78 61 6d 70 6c 65 03 6f 72
+      67 00 00 40 00 01 00 00 06 28 00 1e 00 01 03 64
+      6e 73 07 65 78 61 6d 70 6c 65 03 6f 72 67 00 00
+      01 00 03 02 63 6f 00 0a 00 00
+
+    Resource record (human-readable):
+      _dns.example.org.  1576  IN SVCB 1 dns.example.org (
+          alpn=co docpath )
+
+The root path is RECOMMENDED but not required. Here are two examples where the "docpath" represents
+paths of varying depth. First, "/dns" is provided in the following example
+(the last 8 bytes "00 0a 00 04 63 64 6e 73", "63" marking the CBOR element as a text string of
+length 3, see {{Section 3 of -cbor}}):
+
+    Resource record (binary):
+      04 5f 64 6e 73 07 65 78 61 6d 70 6c 65 03 6f 72
+      67 00 00 40 00 01 00 00 00 55 00 22 00 01 03 64
+      6e 73 07 65 78 61 6d 70 6c 65 03 6f 72 67 00 00
+      01 00 03 02 63 6f 00 0a 00 04 63 64 6e 73
+
+    Resource record (human-readable):
+      _dns.example.org.    85  IN SVCB 1 dns.example.org (
+          alpn=co docpath="dns" )
+
+Second, an examples for the path "/n/s" (the last 8 bytes "00 0a 00 04 61 6e 61 73", "61" marking
```

Comment:
It looks like this presumes a value of 0x000a for the "docpath" registration.  Ideally I think the example would use a value from the reserved range (65280+) until a final value is assigned by IANA.

Also, I believe "61" should be "63" in this example, as it is a string of length 3.